packages/ui-components/src/components/Inputs/InputLabel.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import { InfoOutlined } from '@material-ui/icons';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Tooltip } from '../Tooltip';
+
+/** Styled label for inputs. Handles tooltips for labels if present. */
+
+const LabelWrapper = styled.span`
+  display: flex;
+  align-items: center;
+`;
+
+const TooltipWrapper = styled.span`
```

Comment:
I know that I said not to worry about matching the design for the field grouping but since this is a new component I think we should make it look like the design as much as possible (sorry for the inconsistency).

Reply:
Sure - the only thing is this label component gets used by all the inputs, so if I style it to match the design, it will change all the labels everywhere - is that okay?